tsconfig.base.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "The base configuration for all packages.",
+  "extends": "@vercel/style-guide/typescript",
```

Comment:
Let's make sure @Ethan-Arrowood reviews this since there were changes coming to the style guide:

- https://github.com/vercel/style-guide/pull/60